Source/ACE.Server/WorldObjects/Player_Skills.cs

```diff
@@ -177,8 +177,14 @@ public bool TrainSkill(Skill skill, int creditsSpent)
 
             creatureSkill.AdvancementClass = SkillAdvancementClass.Trained;
             creatureSkill.Ranks = 0;
-            creatureSkill.ExperienceSpent = 0;
-            creatureSkill.InitLevel += 5;
+            if (applyCreationBonusXP)
+            {
+                creatureSkill.ExperienceSpent = 526;
```

Comment:
this is going to mess up all the specialized skills

Reply:
negative, applyCreationBonusXP did not get set true for character creation